chapters/ch12/0.setup.ipynb

```diff
@@ -95,13 +95,121 @@
      "text": [
       "Wiping \"products\" collection\n",
       "Creating \"products\" collection\n",
-      "Status: Success\n",
-      "Adding LTR QParser for products collection\n",
-      "Adding LTR Doc Transformer for products collection\n",
+      "Schema: {\n",
```

Comment:
This is OpenSearch output. Notebook needs to be rerun to generate the Solr output as the default.

Reply:
I've rerun these.

Reply:
There was a log statement there as well.